src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -655,6 +655,154 @@ class ValidatorStore extends BaseStore {
+              }
+            }
+          })
+          .on('transactionHash', (txHash: string) => {
+            applicationStore.txHash = txHash;
+            applicationStore.showTransactionPendingPopUp(true);
+          })
+          .on('error', (error: any) => {
+            // eslint-disable-next-line no-prototype-builtins
+            const isRejected: boolean = error.hasOwnProperty('code');
+            GoogleTagManager.getInstance().sendEvent({
+              category: 'validator_register',
+              action: 'register_tx',
+              label: isRejected ? 'rejected' : 'error',
+            });
+            console.debug('Contract Error', error.message);
+            applicationStore.setIsLoading(false);
+            resolve(false);
+          })
+          .catch((error: any) => {
+            applicationStore.setIsLoading(false);
+            if (error) {
+              notificationsStore.showMessage(error.message, 'error');
+              GoogleTagManager.getInstance().sendEvent({
+                category: 'validator_register',
+                action: 'register_tx',
+                label: 'error',
+              });
+              resolve(false);
+            }
+            console.debug('Contract Error', error);
+            resolve(true);
+          });
+    });
+  }
+
+  async createKeySharePayloadUnsafe(update: boolean = false): Promise<Map<string, any> | false> {
+    update;
+    const ssvStore: SsvStore = this.getStore('SSV');
+    const walletStore: WalletStore = this.getStore('Wallet');
+    const clusterStore: ClusterStore = this.getStore('Cluster');
+    const totalCost = 4.5;
+    try {
+      const amountInWei = ssvStore.prepareSsvAmountToTransfer(walletStore.toWei(totalCost));
+      const payload = this.createPayload(this.keySharePublicKey,
+          this.keySharePayload?.operatorIds.map(Number).sort((a: number, b: number) => a - b),
+          this.keySharePayload?.sharesData, `${amountInWei}`,
+          await clusterStore.getClusterData(clusterStore.getClusterHash(this.keySharePayload?.operatorIds.sort())));
+      return payload;
+    } catch (e: any) {
+      console.log(e.message);
+      return false;
+    }
+  }
+
+  async validateKeySharePayloadUnsafe(): Promise<KeyShareError> {
+    const OK_RESPONSE_ID = 0;
+    const ERROR_RESPONSE_ID = 4;
+    const VALIDATOR_EXIST_ID = 3;
+    const PUBLIC_KEY_ERROR_ID = 5;
+    const keyShares = new KeyShares();
+    const accountStore: AccountStore = this.getStore('Account');
+    const { OK_RESPONSE,
+      CATCH_ERROR_RESPONSE,
+      VALIDATOR_EXIST_RESPONSE,
+      VALIDATOR_PUBLIC_KEY_ERROR,
+    } = translations.VALIDATOR.KEYSHARE_RESPONSE;
+    try {
+      const fileJson = await this.keyShareFile?.text();
+      const operatorStore: OperatorStore = this.getStore('Operator');
+      const walletStore: WalletStore = this.getStore('Wallet');
+      await accountStore.getOwnerNonce(walletStore.accountAddress);
+      const { ownerNonce } = accountStore;
```

Comment:
Please add a console.log with ownerNonce, doesnt have to be at this point but somewhere during the registration flow i would really like to see such a log

`Current registration nonce: ${ownerNonce}`

Reply:
added

Reply:
done